src/wp-content/themes/twentytwentythree/styles/aubergine.json

```diff
@@ -138,7 +133,9 @@
 				"elements": {
 					"link": {
 						"color": {
-							"text": "var(--wp--preset--color--contrast)"
+							"text": {
+								"ref": "styles.color.text"
```

Comment:
`ref` currently isn't included in the theme.json schema, so it will throw a warning in text editors.

There is a PR here to address this: https://github.com/WordPress/gutenberg/pull/45897